contracts/implement/MatchingEngineCore.sol

```diff
@@ -388,7 +389,7 @@ abstract contract PairManagerCore is Block, PairManagerCoreStorage {
         address _trader,
         bool _isBase,
         uint128 _maxFindingWordsIndex
-    ) internal virtual returns (uint256 sizeOut, uint256 openOtherSide) {
+    ) internal virtual returns (uint256 baseOut, uint256 quoteOut) {
```

Comment:
Hi @JustInPosition , I saw you change `sizeOut ` to `baseOut ` and `openOtherSide` to `quoteOut `. In case user open market with base is great, but when an open market with quote asset, the `sizeOut ` is the quote asset and `openOtherSide` is base. So I think we need to discuss finding some names that can use for all cases. Have a nice day.